Sources/NIO/ChannelOption.swift

```diff
@@ -240,7 +273,7 @@ extension ChannelOptions {
 
                 applier(channel)(key, value).map {
                     applyNext()
-                }.cascadeFailure(to: applyPromise)
+                    }.cascadeFailure(to: applyPromise)
```

Comment:
nit: might want to fix the indent, thanking Xcode whilst doing that ;)

Reply:
Done. :grin: